docs/root/modules/sumologic/schema.md

```diff
@@ -0,0 +1,18 @@
+## Sumologic Schema
+
+.. _sumologic_schema:
+
+### Sumologic core platform
+
+Representation of a system sending logs to Sumologic core platform. Be aware that this may vary depending on your environment.
+Default code expects _sourceCategory to be formatted as BU/DC/SOURCE_TYPE aka business unit, datacenter or location, and source type. Source type as SERVER or NXLOG_* to match targeted systems (linux, windows...). Some fields may need to be set through [Field Extraction Rules](https://help.sumologic.com/docs/manage/field-extractions/)
+
+| Field | Description |
+|-------|-------------|
+|tool_first_seen| Timestamp of when first available logs for host is available since first sync|
```

Comment:
I think you meant to add the schema for the hosts?

Reply:
I don't understand the comment.
On tool_first_seen, this is limited by the duration given to sumologic query. if query only last 7d log, the earliest first seen is only 7d old, even if logging platform can have more.
But querying on long time period is expensive. And if the platform aka cartography server lives long enough, it will get there.